packages/datetime/src/dateRangePicker.tsx

```diff
@@ -155,25 +162,30 @@ export class DateRangePicker
 
     public render() {
         const modifiers = combineModifiers(this.modifiers, this.props.modifiers);
-        const { className, locale, localeUtils, maxDate, minDate } = this.props;
+        const { className, isSequential, locale, localeUtils, maxDate, minDate } = this.props;
         const isShowingOneMonth = DateUtils.areSameMonth(this.props.minDate, this.props.maxDate);
         const { leftView, rightView } = this.state;
         const { disabledDays, selectedDays } = this.states;
 
-        if (isShowingOneMonth) {
-            // use the left DayPicker when we only show one
+        if (isSequential || isShowingOneMonth) {
```

Comment:
How extensible is this approach to the use case where consumers may want to show only one month at a time in the `DateRangePicker` (by passing in their own `numberOfMonths` prop, e.g.)? We've gotten this request before from a project where users were getting confused by every variant that involved two display months.